docs/resources/Guild_Scheduled_Event.md

```diff
@@ -6,54 +6,99 @@ A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
+| status                | [event status](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-status)                | the status of the scheduled event                                                                   |
+| entity_type           | [scheduled entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types) | the type of the scheduled event                                                                     |
+| entity_id             | ?snowflake                                                                                                                     | the id of an entity associated with a guild scheduled event                                         |
+| entity_metadata **    | ?[entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata)   | additional metadata for the guild scheduled event                                                   |
+| creator?              | [user](#DOCS_RESOURCES_USER/user-object) object                                                                                | the user that created the scheduled event                                                           |
+| user_count?           | integer                                                                                                                        | the number of users subscribed to the scheduled event                                               |
+
+\* `creator_id` will be null for events created before October 25th, 2021, when the concept of `creator_id` was introduced and tracked
+
+\** See [field requirements by entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-field-requirements-by-entity-type) to understand the relationship between `entity_type` and the following fields: `channel_id`, `entity_metadata`, and `scheduled_end_time` 
 
 ###### Guild Scheduled Event Privacy Level
 
 | Level      | Value | Description                                              |
 | ---------- | ----- | -------------------------------------------------------- |
-| PUBLIC     | 1     | the scheduled event is public and available in discovery |
 | GUILD_ONLY | 2     | the scheduled event is only accessible to guild members  |
 
 ###### Guild Scheduled Event Entity Types
 
 | Type           | Value |
 | -------------- | ----- |
-| NONE           | 0     |
 | STAGE_INSTANCE | 1     |
 | VOICE          | 2     |
 | EXTERNAL       | 3     |
 
+###### Field Requirements By Entity Type
+
+The following matrix describes field requirements based on current entity type.
+
+`non null` : This field is required to be a non-null value
```

Comment:
```suggestion
`non-null` : This field is required to be a non-null value
```

Reply:
Good callout. I'm adding to change log as well, my intention was to draw attention to this temporarily since these docs were only recently added, and then remove the warn blocks at later date so the docs aren't cluttered